draft-ietf-quic-transport.md

```diff
@@ -3868,6 +3896,11 @@ thanks to all.
 
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-quic-transport-09
+
+- Added PATH_CHALLENGE and PATH_RESPONSE frames to replace PING with Data and
+  PONG frame. Changed ACK frame type from 0x0e to 0x0d. (#000)
```

Comment:
Can you open an issue for this, so we can track it?

Reply:
Yes, good point. Filed #1091.